MotionMark/tests/resources/math.js

```diff
@@ -38,19 +41,43 @@ SimpleKalmanEstimator = Utilities.createSubclass(Experiment,
         }
 
         this.estimate = this.estimate + this._gain * (newMeasurement - this.estimate);
-    },
+    }
 
-    reset: function()
+    reset()
     {
-        Experiment.prototype.reset.call(this);
+        super.reset();
         this._initialized = false;
         this.estimate = 0;
     }
-});
+}
+
+class PIDController {
```

Comment:
I think this class should be removed. We have not used it for many years.

Reply:
And it doesn't actually work.